Makefile

```diff
@@ -346,6 +364,32 @@ update-proto-files:
 	@echo 'option swift_prefix = "Proto3";' >> Protos/google/protobuf/unittest_import_public_proto3.proto
 	@echo 'option swift_prefix = "Proto3";' >> Protos/google/protobuf/unittest_proto3.proto
 
+SWIFT_CONFORMANCE_PLUGIN_SOURCES= \
+	Sources/Conformance/conformance.pb.swift \
+	Sources/Conformance/main.swift
+
+$(SWIFT_CONFORMANCE_PLUGIN): $(SWIFT_CONFORMANCE_PLUGIN_SOURCES)
+	${SWIFT} build
+
+# Runs the conformance tests.
+test-conformance: $(SWIFT_CONFORMANCE_PLUGIN) $(CONFORMANCE_HOST) failure_list_swift.txt
+	( \
+		ABS_PBDIR=`cd ${GOOGLE_PROTOBUFS_CHECKOUT}; pwd`; \
+		$${ABS_PBDIR}/conformance/conformance-test-runner --failure_list failure_list_swift.txt $(SWIFT_CONFORMANCE_PLUGIN); \
+	)
+
+# The 'conformance-host' program is part of the protobuf project.
+# It generates test cases, feeds them to our plugin, and verifies the results:
+conformance-host: $(CONFORMANCE_HOST)
+
+$(CONFORMANCE_HOST):
+	( \
+		cd ${GOOGLE_PROTOBUFS_CHECKOUT}; \
```

Comment:
Just noticed this:  The macro here should be `GOOGLE_PROTOBUF_CHECKOUT`  (singular) since `protobuf` is the name of the project on github:  `github.com/google/protobuf/`